### PR TITLE
adding Cloud Network subclasses in IbmPowerVC NetworkManager 

### PR DESCRIPTION
**Pull request by Kuldip-Nanda**

### CODE REVIEW

**Comment by Kuldip-Nanda** [comment]
[WIP]

File: app/models/manageiq/providers/ibm_power_vc/network_manager/security_groub.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ManageIQ::Providers::Openstack::NetworkManager::SecurityGroup.include(ActsAsStiLeafClass)
```
Comment (jaywcarman) [inline app/models/manageiq/providers/ibm_power_vc/network_manager/security_groub.rb:1]: Filename typo, should be `security_group.rb`

File: app/models/manageiq/providers/ibm_power_vc/network_manager.rb [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 class ManageIQ::Providers::IbmPowerVc::NetworkManager < ManageIQ::Providers::Openstack::NetworkManager
   require_nested :EventCatcher
+  require_nested :CloudNetwork
+  require_nested :CloudSubnet
+  require_nested :FloatingIP
+  require_nested :NetworkPort
+  require_nested :NetworkRouter
+  require_nested :SecurityGroup
+
```
Comment (agrare) [inline app/models/manageiq/providers/ibm_power_vc/network_manager.rb:11]: This extra newline isn't necessary

**Review by agrare — APPROVED** [review]

File: app/models/manageiq/providers/ibm_power_vc/network_manager/floating_ip.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+ManageIQ::Providers::Openstack::NetworkManager::FloatingIP.include(ActsAsStiLeafClass)
```
Comment (agrare) [inline app/models/manageiq/providers/ibm_power_vc/network_manager/floating_ip.rb:1]: This is the wrong class name, fixed in https://github.com/ManageIQ/manageiq-providers-ibm_power_vc/pull/55

**Comment by Fryguy** [comment]
We cannot backport this to najdorf because it will require a database migration.

**Pull request merged**